.travis.yml

```diff
@@ -21,63 +21,65 @@ env:
         - BUILD_RUNTIMES=$HOME/.runtimes
 
     matrix:
+        - TASK=check-py36
```

Comment:
I think these should go after check-format.

Rule of thumb is that fast jobs should always run first (unless they're fairly unimportant), and the various linting tasks are much faster than actually running the tests.